Trappist/src/Promact.Trappist.Utility/Constants/StringConstants.cs

```diff
@@ -161,5 +161,15 @@ public string MagicString
             }
         }
         #endregion
+
+        #region "Question Constants"
+        public string QuestionExistInTestError
+        {
+            get
+            {
+                return "Question exist in Test.Question cannot be deleted";
```

Comment:
give one space. after one statement is completed. you can make it like
Question exist in Test. Question can not be deleted.